pydov/util/dovutil.py

```diff
@@ -20,7 +21,9 @@ def get_dov_xml(url):
         The raw XML data of this DOV object as bytes.
 
     """
-    request = requests.get(url, timeout=60)
+    headers = {'user-agent': 'PyDOV/%s' % pydov.__version__}
+
+    request = requests.get(url, headers=headers, timeout=60)
     request.encoding = 'utf-8'
     return request.text.encode('utf8')
 
```

Comment:
Out of curiosity: why the need for the 'user-agent'?

Reply:
By the way, I'm currently running through a lot of the xmls and rest assure, it's still the only one with an error. But it's good to have a check for new files in the future!

Reply:
I added this because I was at it in the code anyway. There is no need. Without it it says 'python-requests' in the logs, now it says 'PyDOV/0.1.0'. Serverside there is no difference between the two (or whatever you would put in the user-agent).

Regarding the invalid XML: we checked in the database and, for gecodeerde lithologie at least, this was the only one with that strange character. Good catch ;)